src/tx/applyTx/checkEpochLength.test.js

```diff
@@ -20,11 +20,14 @@ describe('checkEpochLength', () => {
     const state = getInitialState();
     const epochLength = Tx.epochLength(4);
 
-    checkEpochLength(state, epochLength, {
+    const bridgeState = {
       epochLengths: [4],
```

Comment:
Maybe worth to initialize ``epochLength``?

```
const bridgeState = {
  epochLengths: [4],
  epochLength: 4;
}
```

On a side note - naming two fields with singular and plural forms usually leads to human errors. Maybe we can rename array to ``epochLengthVariants``?

```

Reply:
anyway, don't mind as well. It is only a unit test